talknet/demoTalkNet.py

```
@@ -1,726 +1,726 @@
+	# Extract video between start_seconds and end_seconds and save it to extracted_video.mp4
+	start = start_seconds
+	start_frame = int(start * fps)
+	end_frame = int(end_seconds * fps)
+	duration = end_seconds - start_seconds
+
+	# Extract audio
+	print("Extracting audio...")
+	t = time.time()
+	command = ("ffmpeg -y -i '%s' -ss %s -t %s -qscale:a 0 -ac 1 -vn -threads %d -ar 16000 %s -loglevel panic" % \
+		(video_path, start_seconds, duration, data_loader_thread, audioFilePath))
+	subprocess.call(command, shell=True, stdout=None)
```

Ruff (0.13.3)

`subprocess` call with `shell=True` identified, security issue

(S602)